packages/react-liveness/src/components/FaceLivenessDetector/shared/FaceLivenessErrorModal.tsx

```diff
@@ -116,8 +121,12 @@ export const FaceLivenessErrorModal: React.FC<FaceLivenessErrorModalProps> = (
 
   return (
     <Overlay className={LivenessClassNames.OpaqueOverlay}>
-      <Toast>
-        {children}
+      <Toast
+        aria-labelledby="amplify-liveness-error-heading"
+        aria-describedby="amplify-liveness-error-description"
+        role="alertdialog"
+      >
+        <div id="amplify-liveness-error-description">{children}</div>
```

Comment:
Hmmmm, how does `aria-describedby` work if `children` is not a string?

Reply:
Actually nvm to above we can put the id in the modal. But I think it will just go through any nested elements and read the text, and if there is none anywhere then does nothing. Like here it reads the heading and message
```
<>
      <Flex
        className={LivenessClassNames.ErrorModal}
        id="amplify-liveness-error-description"
      >
        <AlertIcon ariaHidden variation="error" />
        <Text
          className={LivenessClassNames.ErrorModalHeading}
          id="amplify-liveness-error-heading"
        >
          {heading}
        </Text>
      </Flex>
      {message}
    </>
```